client/app/scripts/superdesk-authoring/views/article-edit.html

```diff
@@ -1,114 +1,121 @@
-  <div sd-package data-item="item" class="composite-preview" ng-if="!_editable"></div>
-</div>
-
-<div class="field pull-left" ng-if="item.sign_off">
-    <label translate>Sign-Off</label>
-    <div>{{item.sign_off}}</div>
-</div>
\ No newline at end of file
+<div class="field" ng-class="{'limit-error': item.headline.length > limits.headline}">
+  <label translate>Headline</label>
+    <span sd-character-count data-item="item.headline" data-html="true" data-limit="limits.headline"></span>
+    <div id="title" class="headline"
+        sd-text-editor
+        data-config="{disableToolbar: true, disableReturn: true}"
+        data-language="item.language"
+        ng-model="item.headline"
+        ng-change="autosave(item)"
+        ng-if="_editable"
+        ng-trim="false"></div>
+  <div class="headline" ng-if="!_editable">{{item.headline}}</div>
+</div>
+
+<div class="field" class="abstract" ng-class="{'limit-error': item.abstract.length > limits.abstract}">
+  <label translate>Abstract</label>
+    <span sd-character-count data-item="item.abstract" data-html="true" data-limit="limits.abstract"></span>
+    <div id="abstract" class="abstract"
+        sd-text-editor
+        data-config="{disableToolbar: true}"
+        data-language="item.language"
+        ng-model="item.abstract"
+        ng-change="autosave(item)"
+        ng-if="_editable"
+        ng-trim="false"></div>
+  <div class="abstract" ng-if="!_editable">{{item.abstract}}</div>
+</div>
+
+<div class="field">
+  <label translate>By</label>
+  <input type="text"
+         class="byline"
+         id="byline"
+         ng-model="item.byline"
+         ng-change="autosave(item)"
+         ng-if="_editable">
+  <div ng-if="!_editable">{{item.byline}}</div>
+</div>
+
+<div class="field">
+    <label translate>Dateline</label>
+    <div class="dateline">
+        <div sd-meta-locators
+             class="dateline-city"
+             ng-show="_editable"
+             data-list="metadata.cities"
+             data-item="item"
+             data-fieldprefix="dateline"
+             data-field="located"
+             data-postprocessing="updateDateline(item, city)"
+             data-change="autosave(item)">
+        </div>
+
+        <div class="dateline-date">
+            <select id="datelineMonth" ng-disabled="!item.dateline.located || !_editable" ng-change="resetNumberOfDays(true)" ng-model="datelineMonth">
+                <option value=""></option>
+                <option value="{{monthValue}}" ng-repeat="(monthName, monthValue) in monthNames track by monthName">{{monthName}}</option>
+            </select>
+            <select id="datelineDay" ng-disabled="!item.dateline.located || !_editable" ng-change="modifyDatelineDate()" ng-model="datelineDay">
+                <option value=""></option>
+                <option value="{{day}}" ng-selected="datelineDay === day" ng-repeat="day in daysInMonth track by day">{{day}}</option>

```

Comment:
For both dropdown menus it is better to replace `ng-repeat` with `ng-options`. The latter is [more efficient](https://docs.angularjs.org/api/ng/directive/select) as explained in the linked docs, since it does not create a new scope for every item in the collection.